manubot/process/util.py

```diff
@@ -202,17 +202,17 @@ def get_metadata_and_variables(args):
     return metadata, variables
 
 
-def get_citation_df(args, text):
+def get_citekeys_df(args, text):
     """
-    Generate citation_df and save it to 'citations.tsv'.
-    citation_df is a pandas.DataFrame with the following columns:
-    - manuscript_id: citation ids extracted from the manuscript content files.
-    - detagged_id: manuscript_id but with tag citations dereferenced
-    - standard_id: detagged_id standardized
-    - short_id: standard_id hashed to create a short base
+    Generate citekeys_df and save it to 'citations.tsv'.
```

Comment:
The terminology here seems important.  Do we want to use modifiers of `key` or `citekey`?  The code elsewhere calls some of these entities `*_citekey`.

Reply:
See docstring for `get_citekeys_df`:

```
    Generate citekeys_df and save it to 'citations.tsv'.
    citekeys_df is a pandas.DataFrame with the following columns:
    - manuscript_key: citation keys extracted from the manuscript content files.
    - detagged_key: manuscript_key but with tag citekeys dereferenced
    - standard_key: detagged_key standardized
    - short_key: standard_key hashed to create a shortened citekey
```

Because these are column names within a "citations" file, I thought it'd be okay to just use a `_key` suffix. However, I think you are right that it'd be a bit more explicit to use `_citekey` as the suffix.

Reply:
Done in https://github.com/manubot/manubot/pull/129/commits/cf760acc177e0638ec11379af3ff18474c3c3676